vendor/bat-native-ads/test/data/client.json

```diff
@@ -21,12 +20,7 @@
   "adUUID": "",
   "locale": "en",
   "lastUserActivity": 0,
```

Comment:
can we clear this from existing state or does that happen somewhere that I missed?